frame/support/procedural/src/pallet/expand/hooks.rs

```diff
@@ -40,19 +40,17 @@ pub fn expand_hooks(def: &mut Def) -> proc_macro2::TokenStream {
 		quote::quote! {
 			#frame_support::log::info!(
 				target: #frame_support::LOG_TARGET,
-				"⚠️ {} declares internal migrations (which *might* execute), setting storage version to {:?}",
```

Comment:
is it not possible to still output the current and new storage version?

Reply:
As we don't update the version automatically in the storage, I'm not sure what the benefit of the message is.

Reply:
If possible, it is still benefical to display it.

Reply:
it be a violation of `current_storage_version` and `on_chain_storage_version` are not the same in most cases, I think both should be displayed here.

Reply:
@bkchr ^^

Reply:
seems fixed

Reply:
seems fixed